notifications_utils/formatters.py

```diff
@@ -287,6 +288,83 @@ def normalise_whitespace(value):
     return ' '.join(strip_and_remove_obscure_whitespace(value).split())
 
 
+def get_action_links(html: str) -> list[str]:
+    """Get the action links from the html email body and return them as a list. (insert_action_link helper)"""
+    # set regex to find action link in html, should look like this:
+    # &gt;&gt;<a ...>link_text</a>
+    action_link_regex = re.compile(
+        r'(>|(&gt;)){2}(<a style=".+?" href=".+?"( title=".+?")? target="_blank">)(.*?</a>)'
+    )
+
+    return re.findall(action_link_regex, html)
+
+
+def get_img_link() -> str:
```

Comment:
Suggestion to be "cleaner" but what you have works too 
```
def get_img_link() -> str:
    environment_mapping = {
        'production': 'prod-va-gov-assets',
        'staging': 'staging-va-gov-assets',
        'performance': 'staging-va-gov-assets'
    }
    prefix = environment_mapping.get(os.environ.get('NOTIFY_ENVIRONMENT'), 'dev-va-gov-assets')
    return f'https://{prefix}.s3-us-gov-west-1.amazonaws.com/img/vanotify-action-link.png'
 ```

Reply:
I like it. I'll change it to that 👍